autosklearn/metalearning/input/aslib_simple.py

```diff
@@ -147,9 +151,11 @@ def _read_configurations(self, filename):
                 configuration = dict()
                 algorithm_id = line["idx"]
                 for hp_name, value in line.items():
+                    # Todo adapt to search space
                     if not value or hp_name == "idx":
                         continue
-
+                    if hp_name not in self.cs.get_hyperparameter_names():
```

Comment:
Same here to move it outside the loop. I checked the code and calling the function does some processing.